minecode/tests/test_debian.py

```diff
@@ -332,7 +332,6 @@ def test_parse_packages_from_installed_status(self):
 
 class DebianLSLRTest(BaseDebianTest):
 
-    @expectedFailure
     def test_DebianDirectoryIndexVisitor_from_debian(self):
         uri = 'http://ftp.debian.org/debian/ls-lR.gz'
         test_loc = self.get_test_loc('debian/lslr/ls-lR_debian.gz')
```

Comment:
We think its better to not have the gzipped ls-lR files in the tests, rather we would want to keep the ls-lR files in text form like https://github.com/nexB/purldb/blob/main/minecode/tests/testfiles/directories/ls-lr . Then during testing, we gzip the files and then pass it into `mocked_requests_get()`

We want to do this so it is easier to see what changes have been made to the test data, as you cannot see changes made to a binary file in git.

Reply:
Changes made, both test files are now kept in text form and on running the tests we make a `.gz` file from them at a temp location and then pass it to ```mocked_request_get()```